blade-graphics/src/vulkan/descriptor.rs

```diff
@@ -7,6 +7,7 @@ const COUNT_BASE: u32 = 16;
 #[derive(Debug)]
 pub struct DescriptorPool {
     sub_pools: Vec<vk::DescriptorPool>,
+    stage: usize,
```

Comment:
let's call it something that's not confused with "shader stage"
maybe `growth_iter`?